homework.rb

```diff
@@ -0,0 +1,200 @@
+# 1
+students = [
+    {
+        first_name: 'John',
+        last_name: 'Doe'
+    },
+    {
+        first_name: 'Jane',
+        last_name: 'Doe',
+    },
+    {
+        first_name: 'Jennifer',
+        last_name: 'Aniston',
+    }
+  ]
+  
+new_students = students.map do |students|
```

Comment:
Great work, but the variable's name represents what information the variable contains so change the variable name from ```students``` to ```student```
``` students.each do |student|```